docs/governance.md

```diff
@@ -0,0 +1,200 @@
+
+## Voting
+
+The Thanos project usually runs by informal consensus, however sometimes a formal decision must be made.
+
+Depending on the subject matter, as laid out [above](#decision-making), different methods of voting are used.
+
+For all votes, voting must be open for at least one week. The end date should be clearly stated in the call to vote.
+A vote may be called and closed early if enough votes have come in one way so that further votes cannot change the final decision.
+
+In all cases, all and only [Maintainers](#maintainers-team) are eligible to vote, with the sole exception of the forced removal of a team member, in which said member is not eligible to vote.
+
+Discussion and votes on personnel matters (including but not limited to team membership and maintainership) are held in private on the [thanos-io@googlegroups.com][team]. All other discussion and votes are held in public on the GitHub issues or #thanos-dev CNCF slack channel.
+
+For public discussions, anyone interested is encouraged to participate. Formal power to object or vote is limited to [Maintainers Team](#maintainers-team).
+
+### Governance
+
+It's important for the project to stay independent and focused on shared interest instead of single use case of one company or organization.
+
+We value open source values and freedom, that's why we limit Maintainers Team votes to maximum two from a single organization or company.
+
+We also encourage any other company interested in helping maintaining Thanos to join us to make sure we stay independent.
+
+### Consensus
+
+The default decision making mechanism for the Thanos project is [lazy consensus][lazy]. This means that any decision on technical issues is considered supported by the [team][team] as long as nobody objects.
+
+Silence on any consensus decision is implicit agreement and equivalent to explicit agreement. Explicit agreement may be stated at will.
+
+Consensus decisions can never override or go against the spirit of an earlier explicit vote.
+
+If any [member of Maintainers Team](#maintainers-team) raises objections, the team members work together towards a solution that all involved can accept.
+This solution is again subject to lazy consensus.
+
+In case no consensus can be found, but a decision one way or the other must be made, anyone from [Maintainers Team](#maintainers-team) may call a formal [majority vote](#majority-vote).
+
+### Majority vote
+
+Majority votes must be called explicitly in a separate thread on the appropriate mailing list. The subject must be prefixed with `[VOTE]`.
+In the body, the call to vote must state the proposal being voted on. It should reference any discussion leading up to this point.
+
+Votes may take the form of a single proposal, with the option to vote yes or no, or the form of multiple alternatives.
+
+A vote on a single proposal is considered successful if more vote in favor than against.
```

Comment:
My only (stupid) question: what if we have a even number of members and an equal amount of people vote for/against? (:

Reply:
It has to be more to be successful so in that case it would be declined.

Reply:
Will add to FAQ